### PR TITLE
Remove compatibility for php versions inferior to PHP 8.1

### PR DESCRIPTION
**Pull request by jolelievre**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Starting PrestaShop `9.0` the PHP minimum version will be `8.1` minimum in order to follow the ecosystem and allow us to update some of our dependencies. This PR updates the requirements in the project and cleans the CI since we no longer need to check older PHP versions.
| Type?             | improvement
| Category?         | CO
| BC breaks?        | yes
| Deprecations?     | no
| How to test?      | CI tests green and GA tests green https://github.com/jolelievre/ga.tests.ui.pr/actions/runs/4959060658
| Fixed ticket?     | ~
| Related PRs       | ~
| Sponsor company   | ~

### CODE REVIEW

File: INSTALL.txt [hunk 1]
```diff
@@ -46,7 +46,7 @@
 
 === Requirements
 
-To install PrestaShop 8, you need a web server running PHP 7.2+ and any flavor of MySQL 5.6+ (MySQL, MariaDB, Percona Server, etc.).
+To install PrestaShop 8, you need a web server running PHP 8.1+ and any flavor of MySQL 5.6+ (MySQL, MariaDB, Percona Server, etc.).
```
Comment (lartist) [inline INSTALL.txt]:
```suggestion
To install PrestaShop 9, you need a web server running PHP 8.1+ and any flavor of MySQL 5.6+ (MySQL, MariaDB, Percona Server, etc.).
```

File: README.md [hunk 2]
```diff
@@ -33,7 +32,7 @@ The first stable version of PrestaShop 8.0, was released on October 26th, 2022.
 Server configuration
 --------
 
-To install the latest PrestaShop 8.0, you need a web server running PHP 7.2+ and any flavor of MySQL 5.6+ (MySQL, MariaDB, Percona Server, etc.).
+To install the latest PrestaShop 8.0, you need a web server running PHP 8.1+ and any flavor of MySQL 5.6+ (MySQL, MariaDB, Percona Server, etc.).
```
Comment (lartist) [inline README.md]:
```suggestion
To install the latest PrestaShop 9.0, you need a web server running PHP 8.1+ and any flavor of MySQL 5.6+ (MySQL, MariaDB, Percona Server, etc.).
```

**Review by lartist — DISMISSED** [review]

**Review by FabienPapet — DISMISSED** [review]

**Comment by FabienPapet** [comment]
No qa needed for this as it is only for github actions

**Comment by jolelievre** [comment]
@FabienPapet no it's not, the install process is also impacted as it checks the PHP versions
Although this should be validated by our automatic tests that check the install process

**Review by tleon — APPROVED** [review]

**Review by mflasquin — APPROVED** [review]
Maybe we can try to add 8.2 ? :D

**Review by FabienPapet — APPROVED** [review]

**Comment by jolelievre** [comment]
@mflasquin it's planned for later there is already a pending PR https://github.com/PrestaShop/PrestaShop/pull/29116 but it's blocked by two topics:
- Symfony 5.4 (almost done)
- SwiftMailer replacement

But it's on track don't worry 😉

File: composer.json [hunk 3]
```diff
@@ -55,7 +55,7 @@
         "mrclay/minify": "^3.0.11",
         "nikic/php-parser": "^4.0",
         "nyholm/psr7": "^1.5",
-        "pelago/emogrifier": "^v5.0.1",
+        "pelago/emogrifier": "^v6.0.0",
```
Comment (lartist) [inline composer.json:58]: Why ?

**Review by boherm — APPROVED** [review]

**Pull request merged**